### PR TITLE
Add 'toInlineResponse' to Media

### PR DESCRIPTION
**Pull request by mbardelmeijer**
Follow-up of #1358

### CODE REVIEW

**Comment by rcerljenko** [comment]
@mbardelmeijer @freekmurze  would be great if you can optionally pass a `conversion` name to these functions. eg:
```php
return $media->toResponse($request, 'thumb');
```

and how about responsive images?

File: docs/advanced-usage/outputting-media.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Outputting media
+weight: 5
+---
+
+If you want to output your `$media` instance to the browser, you may use the `toResponse` & `toInlineResponse` methods.
+
+```bash
```
Comment (freekmurze) [inline docs/advanced-usage/outputting-media.md:8]: This should be ```php I guess

**Review by freekmurze — CHANGES_REQUESTED** [review]

**Comment by freekmurze** [comment]
> would be great if you can optionally pass a conversion name to these function

Good idea!, could you take care of this @mbardelmeijer 

Responsive images are out of scope for now.

**Comment by mbardelmeijer** [comment]
@freekmurze @rcerljenko I've checked if it would be possible to add conversion support to these methods, but ran into that the `stream` method from the `Filesystem` doesn't support passing a conversion name. 

We can get the file via the default methods, but then we don't have the memory optimized stream function to use.

We can add a 'getConversionStream' method for instance in the `Filesystem`, and a `conversionStream` to the Media object, but I do think this would grow the scope of the PR too much. Thoughts?

**Comment by rcerljenko** [comment]
@mbardelmeijer yes it adds more work.. let @freekmurze decides

**Comment by freekmurze** [comment]
Let's keep it simple for now.

Thanks @mbardelmeijer

**Pull request merged**